templates/config.json.j2

```diff
@@ -1,25 +1,27 @@
 {
-    "masters": [{% for host in groups['mesosmaster'] %}"{{hostvars[host]['ansible_default_ipv4']['address']}}:5050"{% if not loop.last %},{% endif %}{% endfor %}],
-    "refreshSeconds": 60,
-    "ttl": 60,
+    "zk": "zk://{{ mesos_dns_zookeepers | join(',')}}",
```

Comment:
zk is new configuration variable https://mesosphere.github.io/mesos-dns/docs/configuration-parameters.html